config/CMakeLists.txt

```diff
@@ -78,3 +81,5 @@ install(
   "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}.pc"
   DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig"
 )
+
+include(CheckFortranSourceCompiles)
```

Comment:
Where is this used in the current scope?

Reply:
I went back to the simpler `Findcustom-blas.cmake` and `Findcustom-lapack.cmake` and this is no longer needed